trademapper/js/trademapper.arrows.js

```diff
@@ -399,12 +402,19 @@ define(["d3", "spiralTree", "trademapper.route", "util", "config"], function(d3,
 		return function() { moduleThis.plainMouseOverPath(route); };
 	},
 
+    createPlainMouseOutFunc: function(route) {
```

Comment:
This is unnecessary: because plainMouseOutPath() uses this.highlightedPath and doesn't ever use the route passed to this function, you could remove createPlainMouseOutFunc() and just bind the mouseout event on the line to plainMouseOutPath() instead.